tests/FizzWare.NBuilder.Tests/Unit/ParallelListBuilderTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+using Shouldly;
+using Xunit;
+
+namespace FizzWare.NBuilder.Tests.Unit
+{
+    public class ParallelListBuilderTests
+    {
+        private class MyClass
```

Comment:
Yep, it's easy to just create a simple class for this test. But lets try reusing this [existing class called SimpleClass](https://github.com/nbuilder/nbuilder/blob/master/tests/FizzWare.NBuilder.Tests/TestClasses/SimpleClass.cs).

Reply:
I have updated the test to use `SimpleClass` instead.